Source/CarthageKit/Xcode.swift

```diff
@@ -932,18 +932,18 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 							return false
 						}
 					}
+					.collect()
 					.flatMap(.Concat) { settings -> SignalProducer<TaskEvent<BuildSettings>, CarthageError> in
-						if settings.bitcodeEnabled.value == true {
+						let bitcodeEnabled = settings.reduce(true) { $0 && ($1.bitcodeEnabled.value ?? false) }
+						if bitcodeEnabled {
```

Comment:
This looks correct, it should be sure that all the targets are bitcode-enabled to enable bitcode for the scheme.